drizzlepac/alignimages.py

```diff
@@ -673,35 +703,40 @@ def match_relative_fit(imglist, reference_catalog):
     Parameters
     ----------
     imglist : list
-        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata and source catalogs
+        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata
+        and source catalogs
 
-    reference_catalog : Table
-        Astropy Table of reference sources for this field
+    reference_catalog : astropy.table.Table
+        `~astropy.table.Table` of reference sources for this field
 
     Returns
     --------
     imglist : list
-        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata and source catalogs
+        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata
+        and source catalogs.
 
     """
-    log.info("{} STEP 5b: (match_relative_fit) Cross matching and fitting {}".format("-" * 20, "-" * 27))
+    _log_step('STEP 5b: (match_relative_fit) Cross matching and fitting')
     # 0: Specify matching algorithm to use
-    match = tweakwcs.TPMatch(searchrad=75, separation=0.1, tolerance=2, use2dhist=True)
-    # match = tweakwcs.TPMatch(searchrad=250, separation=0.1,
-    #                          tolerance=100, use2dhist=False)
+    match = tweakwcs.TPMatch(searchrad=75, separation=0.1, tolerance=2,
+                             use2dhist=True)
 
     # Align images and correct WCS
-    # NOTE: this invocation does not use an astrometric catalog. This call allows all the input images to be aligned in
-    # a relative way using the first input image as the reference.
+    # NOTE: this invocation does not use an astrometric catalog. This call
+    # allows all the input images to be aligned in a relative way using the
+    # first input image as the reference.
     # 1: Perform relative alignment
     tweakwcs.align_wcs(imglist, None, match=match, expand_refcat=True)
 
-    # Set all the group_id values to be the same so the various images/chips will be aligned to the astrometric
-    # reference catalog as an ensemble.
-    # astrometric reference catalog as an ensemble. BEWARE: If additional iterations of solutions are to be
-    # done, the group_id values need to be restored.
+    # Set all the group_id values to be the same so the various images/chips
+    # will be aligned to the astrometric reference catalog as an ensemble.
+    # astrometric reference catalog as an ensemble.
+    #
+    # BEWARE: If additional iterations of solutions are to be done,
```

Comment:
If user really have to beware here, the message should go into `log.warning`, no?